test/gui/shared/steps/steps.py

```diff
@@ -516,6 +516,12 @@ def step(context):
         Qt.LeftButton,
     )
     activateItem(waitForObjectItem(names.settings_QMenu, "Force sync now"))
+    try:
+        clickButton(
+            waitForObject(names.remove_All_Files_Remove_all_files_QPushButton, 2000)
```

Comment:
seems a strange try-catch to me, the exception is just passed. but the then should catch errors if something goes wrong.
also, for the usage of static waits, a comment would be reasonable. will `1000` work or is `2000` enough? these types of confusion will not arise if there is already an explanation.

Reply:
The `2000` is the timeout. I put it inside try-catch because it is not guranteed that the object will appear during the force sync. And if it does not appear then there is loockup error. But if we get loockup error, then we want to move on to next step. Because it implies that the message (object) did not appear.